packages/paste-core/components/tooltip/src/index.tsx

```diff
@@ -49,7 +49,7 @@ const Tooltip = React.forwardRef<HTMLDivElement, TooltipProps>(({baseId, childre
   return (
     <>
       {React.Children.only(
-        <TooltipPrimitiveReference {...tooltip} ref={ref}>
+        <TooltipPrimitiveReference {...tooltip} ref={ref} {...children.props}>
```

Comment:
I'm not sure we should destructure `children.props`, because I've never seen that done before. It may be undocumented API and unsafe.  Got any resources that mention this approach I can review?

Reply:
I got it directly from the Reakit docs example https://reakit.io/docs/tooltip/#abstracting

I don't think it's anything fancy. React children have props. If you're inside React.Children.only, there is only one child. So, props.children.props returns the props of the child you passed it.

Reply:
Interesting - I guess it makes sense if we're checking for `React.Children.only`.